boot-qemu.sh

```diff
@@ -126,6 +126,13 @@ function get_full_kernel_path() {
     [[ -f ${KERNEL} ]] || die "${KERNEL} does not exist!"
 }
 
+# Print QEMU version as a five or six digit number
+function get_qemu_ver_code() {
+    QEMU_VER=$("${QEMU[@]}" --version | head -1 | cut -d ' ' -f 4)
+    IFS=. read -ra QEMU_VER <<<"${QEMU_VER}"
+    printf "%d%02d%02d" "${QEMU_VER[@]}"
```

Comment:
yuck

Reply:
I could spell this out as:

```
printf "%d%02d%02d" "${VER_CODE[0]}" "${VER_CODE[1]}" "${VER_CODE[2]}"
```

?

Reply:
We'd still need the `read` though?

Reply:
Correct, I don't think I can use `printf` without splitting the version somehow. This was the cleanest way I could think of.